packages/server/graphql/mutations/helpers/removeFromOrg.ts

```diff
@@ -55,8 +57,43 @@ const removeFromOrg = async (
   }).run()
 
   // need to make sure the org doc is updated before adjusting this
-  const {joinedAt, newUserUntil} = organizationUser
+  const {joinedAt, newUserUntil, role} = organizationUser
   const prorationDate = newUserUntil >= now ? new Date(joinedAt) : undefined
+  if (role === OrgUserRole.BILLING_LEADER) {
+    const organization = await r
+      .table('Organization')
+      .get(orgId)
+      .run()
+    if (organization.tier !== TierEnum.personal) {
```

Comment:
@ackernaut check the biz logic here for account deletions of pro accounts & single-user orgs